src/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  border: 1px solid var(--color-light);
+  border-radius: var(--main-radius);
+  padding: 32px 16px 15px;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;

```

Comment:
let's reset margins for needed elements (p or ul or body) in reset.css file
and set margin-top or margin-bottom for all card elements